app/controllers/api/v2/registration_tokens_controller.rb

```diff
@@ -36,7 +36,8 @@ def invalidate_jwt
           raise ::Foreman::Exception.new(N_("No record found for %s"), params[:id])
         end
         @user.jwt_secret&.destroy
-        process_success _("Successfully invalidated registration tokens for %s.\n" % @user.login)
+        login = @user.login
+        render :json => {:status => 'success', :message => _("Successfully invalidated registration tokens for %s." % login), :user => login}, :status => :ok
```

Comment:
```suggestion
        render :json => { :message => _("Successfully invalidated registration tokens for %s." % login), :user => login }, :status => :ok
```

There is no value in `:status => 'success'` entry: it's more or less duplicate of the `200 OK` status.

Also, having both the success message and `:user => login` seems like redundance of the information. I'd suggest either not include logins in the message or drop the `:user` entry.